src/web/app/components/question-response-panel/question-response-panel.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Input, OnInit } from '@angular/core';
+import {
+  FeedbackQuestionType,
+  FeedbackSession,
+  FeedbackSessionPublishStatus,
+  FeedbackSessionSubmissionStatus,
+  QuestionOutput,
+  ResponseVisibleSetting,
+  SessionVisibleSetting,
+} from '../../../types/api-output';
+import { FeedbackVisibilityType, Intent } from '../../../types/api-request';
+
+/**
+ * Displaying the question response panel.
+ */
+@Component({
+  selector: 'tm-question-response-panel',
+  templateUrl: './question-response-panel.component.html',
+  styleUrls: ['./question-response-panel.component.scss'],
+})
+export class QuestionResponsePanelComponent implements OnInit {
+
+  RESPONSE_HIDDEN_QUESTIONS: FeedbackQuestionType[] = [
```

Comment:
It was not done before, but this variable can be `readonly`

Reply:
fixed in f4952ca